build.gradle

```diff
@@ -43,7 +43,7 @@ ext {
   groovyConsoleExtraDependency = [
     "org.codehaus.groovy:groovy-console:${groovyVersion}"
   ]
-  maxGroovyVersion = snapshotVersion ? "3.9.99" : maxGroovyVersion
+  maxGroovyVersion = snapshotVersion ? "4.9.99" : maxGroovyVersion
```

Comment:
Why restrict it at all for snaptshot versions then?

Reply:
Good question. There is a chance to get strange errors with Groovy 4 or 5, but probably not many people plays with snapshots and alpha Groovy versions. I'm up to relax it. Looking at the implementation settings `9.9.99` would be probably the easiest.

@leonard84 WDYT?